modules/auth/user_form.go

```diff
@@ -41,6 +41,8 @@ type InstallForm struct {
 	OfflineMode                    bool
 	DisableGravatar                bool
 	EnableFederatedAvatar          bool
+	EnableOpenIDSignIn             bool
+	DisableOpenIDRegistration      bool
```

Comment:
I'd rather have these both as `EnableFoo` since it makes more sense when reading